lib/Transforms/Instrumentation/KleeInternalize.cc

```diff
@@ -270,7 +274,22 @@ namespace
           CallInst *ninst = nullptr;
           Builder.SetInsertPoint (&inst);
           
-          if (fn->getName().equals ("verifier.assume"))
+          if (shouldInternalize(*fn))
+          {
+            int id = m_functionId[fn->getName()];
+            Function * newFn = cast<Function> (m_module->getOrInsertFunction(
+              fn->getName().str() + "_" +std::to_string(id),
```

Comment:
use a period instead of underscore to prevent clashes with user defined functions.